ws.py

```diff
@@ -91,6 +92,13 @@
 if not GlobalVars.metasmoke_ws_host:
     log('info', "No metasmoke websocket host found, which is okay if you're anti-websocket")
 
+# Initiate DNS
+if GlobalVars.config.dns_nameservers != 'system':
```

Comment:
In this line and the next one, `GlobalVars.config.dns_nameservers` should be `GlobalVars.dns_nameservers`.

With it as it is currently, in a brief test with an SD instance using a config file unmodified from earlier testing (i.e. with none of the new config entries), this line results in a:
```
AttributeError: 'SectionProxy' object has no attribute 'dns_nameservers'
2021-06-19 01:51:41.260326 UTC
  File "ws.py", line 96, in <module>
    if GlobalVars.config.dns_nameservers != 'system':
```

Reply:
Thats because I made an oopsies, it'd just be GlobalVars.dns_nameservers